pyatlan/client/constants.py

```diff
@@ -23,6 +24,26 @@
 CREATE_GROUP = API(GROUP_API, HTTPMethod.POST, HTTPStatus.OK)
 UPDATE_GROUP = API(GROUP_API, HTTPMethod.POST, HTTPStatus.OK)
 DELETE_GROUP = API(GROUP_API + "/{group_guid}/delete", HTTPMethod.POST, HTTPStatus.OK)
+GET_GROUP_MEMBERS = API(
+    GROUP_API + "/{group_guid}/members", HTTPMethod.GET, HTTPStatus.OK
+)
+REMOVE_USERS_FROM_GROUP = API(
+    GROUP_API + "/{group_guid}/members/remove", HTTPMethod.POST, HTTPStatus.OK
+)
+
+# User APIs
+GET_USERS = API(USER_API, HTTPMethod.GET, HTTPStatus.OK)
+CREATE_USERS = API(USER_API, HTTPMethod.POST, HTTPStatus.OK)
+UPDATE_USER = API(USER_API, HTTPMethod.POST, HTTPStatus.OK)
+DELETE_USER = API(USER_API + "/{user_guid}/delete", HTTPMethod.POST, HTTPStatus.OK)
+GET_USER_GROUPS = API(USER_API + "/{user_guid}/groups", HTTPMethod.GET, HTTPStatus.OK)
+ADD_USER_TO_GROUPS = API(
+    USER_API + "/{user_guid}/groups", HTTPMethod.POST, HTTPStatus.OK
+)
+CHANGE_USER_ROLE = API(
+    USER_API + "/{user_guid}/roles/update", HTTPMethod.POST, HTTPStatus.OK
+)
+GET_CURRENT_USER = API(USER_API + "/current", HTTPMethod.GET, HTTPStatus.OK)
```

Comment:
Yeah, thought about that but then thought I'd keep it consistent with the line immediately above (since f-string won't work there due to actually needing to retain the `{}` as part of the string for parameter replacement) 🤷 — will change it, tho.